packages/drivers/tinylicious-driver/src/insecureTinyliciousTokenProvider.ts

```diff
@@ -30,15 +30,15 @@ export class InsecureTinyliciousTokenProvider implements ITokenProvider {
 
     private getSignedToken(
         tenantId: string,
-        documentId: string,
+        documentId: string | undefined,
         lifetime: number = 60 * 60,
         ver: string = "1.0"): string {
         // Current time in seconds
         const now = Math.round((new Date()).getTime() / 1000);
         const user = { id: uuid(), name: getRandomName() };
 
         const claims: ITokenClaims = {
-            documentId,
+            documentId: documentId ?? "",
```

Comment:
Is Tinylicious also updated with the server change?

Reply:
Yup! Added here: #7104

Reply:
The new driver won't be backward compatible. Should we mention it in the breaking doc?

Reply:
Let's add this to BREAKING.md. (Y)

Reply:
will be added in a separate pr